Gemfile.lock

```diff
@@ -162,76 +162,77 @@ GEM
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rack-timeout (0.6.0)
-    rails (6.0.2.2)
-      actioncable (= 6.0.2.2)
-      actionmailbox (= 6.0.2.2)
-      actionmailer (= 6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actiontext (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    rails (6.0.3.4)
```

Comment:
I'm generally fine with bumping the Rails minor version. I think it deserves its own separate PR if that's what you'd like to do

Reply:
I didn't realize I had bumped this up. What would be the best way to avoid this in the future? Change the rails version that I'm running locally?

Reply:
The Gemfile.lock and yarn.lock files exist so that we can make sure we're all using the same packages even if our machines aren't configured exactly the same. I imagine some command you ran updated them. There are several commands that could do this, so it's hard to say exactly and may not be worth your time to track it down right now

See if you can revert the changes to these lock files and still get the application to run in your local environment. You should be able to do it using only instructions listed in https://github.com/rubyforgood/mutual-aid/blob/main/doc/setup.md without any changes to these files

 If that doesn't work for you, let me know and we can troubleshoot the problems you're having tomorrow